data/wesionary-team.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "wesionaryTEAM Co. Ltd.",
+    "establishedAt": 1580667300,
+    "location": "KVD Complex 2F, Balkumari - 08, Lalitpur",
+    "website": "https://en.wesionary.team/",
+    "email":"hr@wesionary.team",
+    "phone":"",
```

Comment:
```suggestion
    "phone": "",
```

Reply:
Phone number according to : https://www.facebook.com/wesionaryteam/ : +977 9823507282